group-size-refinement-list/react-instantsearch/src/lib/GroupSizeRefinementList.js

```diff
@@ -0,0 +1,108 @@
+import React, { Component } from "react";
+import { Highlight, connectRefinementList } from "react-instantsearch-dom";
+
+import "./GroupSizeRefinementList.css";
+
+class GroupSizeRefinementList extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            expanded: !this.props.showMore,
+            nbGroups: this.props.nbGroups ? this.props.nbGroups : 1
+        };
+    }
+
+    render() {
+        let inclusionArrays = [];
+        const patterns = [...this.props.patterns, /.*/im]
+        let nbGroups = patterns.length
+        let sizeGroups = patterns.reduce((sizeGroups, regex) => {
```

Comment:
```suggestion
        let sizeGroups = patterns.reduce((acc, regex) => {
```

best not to override the variable from the outer scope